docs/development.md

```diff
@@ -3,6 +3,32 @@
 This document explains how to get started with developing for NGINX Ingress controller.
 It includes how to build, test, and release ingress controllers.
 
+## Quick Start
+
+### Initial developer environment build
+
+If you are using **MacOS** and deploying to **minikube**, the following command will build the local nginx controller container image and deploy the ingress controller onto a minikube cluster:
+
+```
+$ make dev-env
+```
+
+**IMPORTANT**: This will uninstall minikube from your machine and reinstall version 0.25.0. 
```

Comment:
Do we absolutely need to enforce this? RBAC can happily coexist with the `AlwaysAllow` authorization mode and be enabled for auditing only. I believe most developers want to keep their local environment up to date.

Reply:
Maybe that (quite unknown) feature would be less destructive? https://github.com/kubernetes/minikube/issues/2371

Reply:
@antoineco profiles do not work with `--vm-driver=kvm2` :)

Reply:
It's only an issue because of the e2e-tests failing. This won't be an issue once @aledbf makes the updates mentioned above. I can remove this part of the script and then deploy with RBAC instead.